### ISSUE TITLE
Add feature to filter search by release_time/creation_height

### ISSUE THREAD

**Issue by Pigges** [OdyseeTeam/odysee-frontend#2543]
As of right now, you can only get 1000 results for a search.
This will miss a lot of claims if there are more than that.
Being able to filter by release_time or creation_height when not available would make it a lot easier to search for content.
This is something that is already implemented in the SDK, so there just needs to be a filter option on the site.

**Comment by tzarebczan** [comment]
Thanks for the feedback. Can you give me an example of something you'd filter by? i.e. in last x days, in last x months, older than x months? 

For now, you can sort descending to view all results.

**Comment by shoxie007** [comment]
> Thanks for the feedback. Can you give me an example of something you'd filter by? i.e. in last x days, in last x months, older than x months?
> 
> For now, you can sort descending to view all results.

Firstly, why does this limit if 1000 exist in the first place? Do API calls tax the infrastructure and servers too much? If so, can you at least remove the limit for premium subscribers to Odysee? This limit has been in place for years. I thought it would be temporary, but obviously I was wrong. 

On Youtube one can search for videos between two dates using a search string in the following format in the search box for the channel:
```after:2020-02-03 before:2020-05-03```

If Odysee search results are restricted to a maximum of 1000, the above would offer a way for users to access videos which fall beyond this bound. Incidentally, Youtube search results are restricted to a maximum of 20,000 I believe. Even if you go to the page/tab which shows all videos ever uploaded, if there are more than 20,000, those won't show. But the above search method offers a way to access videos beyond index 20,000.

**Comment by tzarebczan** [comment]
This limitation is on the LBRY-SDK side, which odysee utilizes. There's an existing issue to research performance impact and remove it. Regardless, still would be good to have this date search feature on Odysee.

**Comment by shoxie007** [comment]
> This limitation is on the LBRY-SDK side, which odysee utilizes. There's an existing issue to research performance impact and remove it. Regardless, still would be good to have this date search feature on Odysee.

Yes, regardless of what happens on the LBRY-SDK end, at least offer users a way to be able to access older videos on a channel with lots and lots of videos (>1000). This feature will be useful irrespective of whether the limitation is removed in LBRY-SDK.

**Comment by Pigges** [comment]
The ability to filter it by a date interval would probably be the most useful option. Or even following on how the SDK implemented it, by having a list of dates and using equal constraints for it.

**Comment by belikor** [comment]
For reference, the previous issue that I found was this https://github.com/lbryio/lbry-sdk/issues/3365

Since then the `wallet/server` code was removed. I suppose it's now in the `hub` code:
https://github.com/lbryio/hub/blob/c0766f6abcb94bd09a79b102bd841be976292262/hub/herald/search.py#L195-L210
